Earthfile

```diff
@@ -13,7 +18,7 @@ test:
     ARG LABEL_FILTER=
     ENV CGO_ENABLED=1
     # Some test require the docker sock exposed
-    WITH DOCKER
+    WITH DOCKER --load enki-image=(+enki-image)
```

Comment:
NOTE: We can do this trick on `kairos-io/kairos` too in order to load the `+base-image` into the target in which we are planning to run enki on.